CryptoBinPkg/CryptoBinPkg.dsc

```diff
@@ -118,7 +118,7 @@
   TimerLib|MdePkg/Library/BaseTimerLibNullTemplate/BaseTimerLibNullTemplate.inf
   UefiRuntimeServicesTableLib|MdePkg/Library/UefiRuntimeServicesTableLib/UefiRuntimeServicesTableLib.inf  #???
   IoLib|MdePkg/Library/BaseIoLibIntrinsic/BaseIoLibIntrinsic.inf                                          #???
-  OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLib.inf
+  OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLibFull.inf
```

Comment:
Since the main thing the full instance brings is ECC support and can increase overall size, if this is only needed for DXE linked modules (which I'd suspect), can you leave `OpensslLib.inf` under this generic section and add `OpensslLibFull.inf` under the `[LibraryClasses.common.DXE_DRIVER, LibraryClasses.common.UEFI_APPLICATION]` section?

Reply:
That makes sense.  Updated.